packages/lesswrong/components/dialogues/DialoguesList.tsx

```diff
@@ -140,7 +140,7 @@ const DialoguesList = ({ classes }: { classes: ClassesType }) => {
   const [showOptIn, setShowOptIn] = useState(optInStartState);
 
   const { results: dialoguePosts } = usePaginatedResolver({
-    fragmentName: "PostsPage",
+    fragmentName: "PostsList",
```

Comment:
should these also be `PostsListWithVotes`?

Reply:
Yes!